src/main/java/hw3/ex2/ElementMetalColorComposite.java

```diff
@@ -0,0 +1,48 @@
+package hw3.ex2;
+
+import hw3.compositepages.AbstractPageComposite;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ElementMetalColorComposite extends AbstractPageComposite {
+    @FindBy(xpath = "//label[normalize-space()='Water']")
+    private WebElement checkBoxWater;
+
+    @FindBy(xpath = "//label[normalize-space()='Wind']")
+    private WebElement checkBoxWind;
+
+    @FindBy(xpath = "//label[normalize-space()='Selen']" )
+    private WebElement radioSelen;
```

Comment:
It will be better get list checkboxes and list radiobuttons. Name of the button should be send as parameter